.githooks/pre-commit

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+function dockercompose {
+  # use docker-compose then fallback to new docker compose
+  docker-compose  "$@"
+  docker compose  "$@"
+}
+
+echo "run Flake8..."
+
+dockercompose run -T --rm web flake8 geotrek
```

Comment:
How about checking changed files only?

Reply:
we don't check changed files, we check all files